packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,450 @@
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/menu';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ActionButton} from '@react-spectrum/button'
+import Copy from '@spectrum-icons/workflow/Copy';
+import Cut from '@spectrum-icons/workflow/Cut';
+import {Item, Menu, MenuTrigger, Section} from '@react-spectrum/menu';
+import Paste from '@spectrum-icons/workflow/Paste';
+```
+
+---
+category: Menus
+---
+
+# MenuTrigger
+
+<p>
+  {docs.exports.MenuTrigger.description}
+  <br />
+  {docs.exports.Menu.description}
+</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Menu', 'MenuTrigger']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/popovers/'}
+  ]} />
+
+## Example
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu>
+    <Item>Aardvark</Item>
+    <Item>Kangaroo</Item>
+    <Item>Snake</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+## Content
+
+The MenuTrigger accepts exactly two children: the Menu and the element which
+triggers the opening of the Menu. The trigger must be the first child passed into
+the MenuTrigger and should be an element that supports press events.
+
+If the Menu is open it will close on blur or scroll events.
+
+### Value
+
+The Menu's selected Item is propagated via the event [`onAction`](#onaction).
+
+The `defaultSelectedKeys` prop can be used to preselect Menu Items in the Menu
+placing it in an uncontrolled state. Alternatively, the `selectedKeys` preselect
+items in the Menu placing it in a controlled state.
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Controlled
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    selectedKeys={['Kangaroo']}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Uncontrolled
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    defaultSelectedKeys={['Snake']}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+The `selectionMode` prop can be used to switch from the default selection of a single Menu Item to multiple Menu Items or none.
```

Comment:
IMO this makes it feel as if the `selectionMode` prop selects a MenuItem by default. 
How about something along the lines of "The `selectionMode` prop specifies how many MenuItems can be selected, ranging from a single MenuItem (default), to multiple MenuItems, or disabling selection entirely."

Reply:
Would be good to write the type for "selectionMode" as well so it gets explained in the prop table as well

Reply:
I didn't document any of the props that came from shared interfaces.

Reply:
That's fine, mind making a ticket or putting a todo comment here then?